index.html

```diff
@@ -0,0 +1,226 @@
+                me.</p>
+            <a href="#">Let's connect</a>
+            <ul>
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/linkedin.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angelist"></a></li>
+            </ul>
+            <div id="resume">
+                <button type="button" class="btn">Get My Resume</button>
+            </div>
+        </div>
+
+        <div class="skill-set">
+            <div class="language">
+                <div class="skill-info">
+                    <h3>Languages</h3>
+                    <img src="images/down-arrow.png" />
+                </div>
+
+                <ul class="skill-detail">
+                    <li>
+                        <img src="images/js.png" />
+                        <h3>JavaScript</h3>
+                    </li>
+                    <li>
+                        <img src="images/html.png" />
+                        <h3>HTML</h3>
+                    </li>
+                    <li>
+                        <img src="images/css.png" />
+                        <h3>CSS</h3>
+                    </li>
+                </ul>
+            </div>
+
+            <div class="framework">
+                <div class="skill-info border-bot">
+                    <h3>Frameworks</h3>
+                    <img src="images/right-arrow.png" />
+                </div>
+
+            </div>
+            <div class="skill">
+                <div class="skill-info border-bot">
+                    <h3>Skills</h3>
+                    <img src="images/right-arrow.png" />
+                </div>
+
+            </div>
+        </div>
+
+    </section>
+
+    <footer>
+        <div id="contact-form">
+            <div class="form-container">
+                <h1>Contact me</h1>
+                <p>If you have an application you are interested in developing, a feature that you need built or a
+                    project
+                    that
+                    needs coding. I’d love to help with it </p>
+                <form action="https://formspree.io/f/xoqbdrjg" method="post" class="form-container form-padding">
+                    <input type="text" name="fullname" maxlength="30" placeholder="Enter your fullname" required />
+                    <input type="email" name="email" placeholder="Enter your email" required />
+                    <textarea placeholder="Write your message here" name="message" maxlength="500" required></textarea>
+                    <div class="submit-btn">
+                        <button class="btn">Get In Touch</button>
+                    </div>
+                </form>
+            </div>
+        </div>
+
+    </footer>
+</body>
```

Comment:
- [ ] Also you need to include the javascript file in your HTML file so the javascript file can work correctly.

```html
    </footer>
    <script src="/main.js"></script>
</body>
```